api/service/stagedstreamsync/downloaders.go

```diff
@@ -59,14 +59,6 @@ func (ds *Downloaders) Close() {
 	}
 }
 
-// DownloadAsync triggers a download
-func (ds *Downloaders) DownloadAsync(shardID uint32) {
```

Comment:
Please keep this function. We may use it on state sync.

Reply:
You can add this function when you will use it. This function inherits interface, but does nothing.

Reply:
Please keep this function. I will remove it if it is not needed.